airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -131,22 +131,39 @@ def get_conn(self) -> api.Salesforce:
         if not self.conn:
```

Comment:
using a cached_property instead is a common pattern in airflow and IMHO cleaner

Reply:
that would be better. I have now switched to a cached_property for self.conn